pkg/management/component_test.go

```diff
@@ -1281,8 +1281,8 @@ func TestExecuteActionsEmail(t *testing.T) {
 	var accessToken = "TOKEN=="
 	var realmName = "master"
 	var userID = "1245-7854-8963"
-	var reqActions = []api.RequiredAction{"action1", "action2"}
-	var actions = []string{"action1", "action2"}
+	var reqActions = []api.RequiredAction{"sms-password-set", "action1", "action2"}
```

Comment:
If you move init_password_action declaration in a const (...) block at the beginning of component.go, you could reuse it here and prevent potential mis-spelling.

Reply:
Done